src/index.ts

```diff
@@ -163,17 +147,30 @@ export default runExtension(async ({ extensionAPI, extension }) => {
     })),
   });
 
-  FEATURES.forEach(({ id, module, defaultEnabled }) => {
+  const flags: boolean[] = [];
+
+  FEATURES.forEach(({ id, module }) => {
     const flag = extensionAPI.settings.get(id);
     const unset = typeof flag === "undefined" || flag === null;
-    if (unset) extensionAPI.settings.set(id, defaultEnabled);
-    module.toggleFeature(
-      unset ? defaultEnabled : (flag as boolean),
-      extensionAPI
-    );
+    if (unset) extensionAPI.settings.set(id, false);
+    flags.push(unset || flag === false ? false : (flag as boolean));
+    module.toggleFeature(unset ? false : (flag as boolean), extensionAPI);
   });
 
+  if (!flags.some((flag) => flag)) {
+    renderToast({
+      id: "roamjs-workbench-features-disabled",
+      // code format below intentional to preserve newlines
+      content: `WorkBench features are **disabled** by default.
+
+Enable them in the WorkBench settings tab!
+      `,
+    });
+  }
+
```

Comment:
This is the important part of this PR. (removing defaultEnabled, adding toast)


Also, I'm interested if ln 155 is even required still.
We create the panel settings with `extensionAPI.settings.panel.create.settings`.
And based on the [roam example](https://github.com/panterarocks49/settings-panel-example/blob/main/extension.js), they create switches without a setting a toggle state, and in my test it seems the default is `off`.

Reply:
If we default to off, it wouldn't be required